ramalama.py

```diff
@@ -29,23 +33,22 @@ def main(args):
             if args[0] == "--dryrun":
                 args.pop(0)
                 dryrun = True
-            elif args[0] in ramalama.funcDict:
+                continue
+
+            if args[0] in ramalama.funcDict:
                 break
-            else:
-                ramalama.perror(f"Error: unrecognized command `{args[0]}`\n")
-                ramalama.usage()
+
+            ramalama.perror(f"Error: unrecognized command `{args[0]}`\n")
+            ramalama.usage(1)
 
         port = "8080"
         host = os.getenv('RAMALAMA_HOST', port)
         if host != port:
             port = host.rsplit(':', 1)[1]
 
-        syspath = '/usr/share/ramalama/python'
-        sys.path.insert(0, syspath)
-
         if conman:
             home = os.path.expanduser('~')
-            wd = "ramalama"
+            wd = "./ramalama"
```

Comment:
I think there's a bug on line 53, I would expect wd in that case to become:

/usr/lib/python3.12/site-packagesramalama

line 53 should be:

target = os.path.join(p, "ramalama")

Reply:
probably won't need this change then